specification/cognitiveservices/data-plane/SpeakerRecognition/Verification/stable/2021-09-05/Verification.json

```
@@ -0,0 +1,1148 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Speaker Verification API",
+    "version": "2021-09-05",
+    "x-ms-code-generation-settings": {
+      "header": "MIT",
+      "strictSpecAdherence": false
+    },
+    "description": "The Azure Cognitive Service Speaker Recognition service provides algorithms that verify and identify speakers by their unique voice characteristics.\nSpeaker Recognition is used to answer the question \"who is speaking?\""
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}/speaker/verification",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "$ref": "#/parameters/endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "parameters": {
+    "endpoint": {
+      "name": "endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    },
+    "apiVersionParam": {
+      "name": "api-version",
+      "x-ms-parameter-location": "client",
+      "x-ms-client-name": "version",
+      "in": "query",
+      "required": false,
```

`api-version` should be required.
```suggestion
      "required": true,
```

---

done